macos-certificate-truster/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "macos-certificate-truster"
+license = "LGPL-3.0-or-later"
```

Comment:
Is this an intentional choice? I'd like to keep everything MIT if possible.

Reply:
sure! I looked at [windows redirector](https://github.com/mitmproxy/mitmproxy_rs/blob/main/windows-redirector/Cargo.toml#L3) and thought it was the right thing

Reply:
Ah yes- the Windows redirector is LGPL because WinDivert and https://crates.io/crates/windivert are LGPL.